packages/styles/src/variables/_type.scss

```diff
@@ -58,9 +58,9 @@ $line-height-sm: 1;
 $line-height-copy: 1.5;
 $line-height-heading: 1.2;
 
-$font-weight-light: 300 !default; // Design System -> Frutiger Neue Light
-$font-weight-normal: 400 !default; // Design System -> Frutiger Neue Roman
-$font-weight-bold: 700 !default; // Design System -> Frutiger Neue Bold
+$font-weight-light: 300 !default;
+$font-weight-normal: 400 !default;
+$font-weight-bold: 700 !default;
```

Comment:
What about `$font-weight-black`?
And since we remove the light font variant, we should also remove the variable `$font-weight-light`.